site/gatsby-site/cypress/e2e/integration/languageSwitcher.cy.js

```diff
@@ -13,18 +13,19 @@ describe('The Language switcher', () => {
     cy.contains('h1', 'AI Incident Database').should('be.visible');
   });
 
-  it.skip('Should update the path with the selected language', () => {
+  it('Should update the path with the selected language', () => {
     cy.visit('/');
 
+    cy.waitForStableDOM();
+
     cy.get('[data-cy="language-switcher"]').click();
 
+    cy.waitForStableDOM();
+
     cy.get('[data-cy="language-switcher"]').contains('[role="button"]', 'Spanish').click();
 
     cy.url().should('contain', '/es/');
 
-    cy.contains(
-      'h1',
-      'Bienvenido a la base de datos de incidentes de Inteligencia Artificial'
-    ).should('be.visible');
+    cy.contains('h1', 'Bienvenido ala base de datos de incidentes de IA').should('be.visible');
```

Comment:
This test should fail because of the typo `ala`. Please fix the typo

Reply:
Hah no that's a weird consequence of using to tags for each part of the title, adding that space breaks the test 😆